arroyo/processing/strategies/filter.py

```diff
@@ -80,7 +83,14 @@ def submit(
                 self.__uncommitted_offsets.pop(partition, None)
             self.__next_step.submit(message)
         elif self.__commit_policy_state is not None:
+            # first scenario, where a CommitPolicy is passed in
+            # and messages are dropped
             self.__uncommitted_offsets.update(message.committable)
+            self.__metrics.increment("arroyo.strategies.filter_step.dropped_messages")
```

Comment:
Can we make this:
```suggestion
            self.__metrics.increment("arroyo.strategies.filter.dropped_messages")
```
It kinda bugs me that a couple of the steps are suffixed with "step" and I think we should standardise it.

Reply:
Done